ci/e2e-tests/test-run.sh

```diff
@@ -603,14 +603,14 @@ case "$OS" in
         # az vm image list --all \
         #     --publisher 'Canonical' --offer '0001-com-ubuntu-server-focal' --sku '20' \
         #     --query "[?publisher == 'Canonical' && offer == '0001-com-ubuntu-server-focal'].{ sku: sku, version: version, urn: urn }" --output table
-        vm_image='Canonical:0001-com-ubuntu-server-focal:20_04-lts-gen2:20.04.202209130'
+        vm_image='Canonical:0001-com-ubuntu-server-focal:20_04-lts-gen2:20.04.202210180'
```

Comment:
Question for @arsing: Why are we pinning to a particular version of the VM image, as opposed to using `...:latest`?

Reply:
Ah, TIL.

https://learn.microsoft.com/en-us/azure/virtual-machines/linux/cli-ps-findimage

>These values can be passed individually or as an image URN, combining the values separated by the colon (:). For example: Publisher:Offer:Sku:Version. You can replace the version number in the URN with latest to use the latest version of the image.

Yes, we should switch to that then.

Reply:
Updated all of them to use latest